networking_nsxv3/api/rpc.py

```diff
@@ -108,90 +81,46 @@ class NSXv3ServerRpcApi(object):
 
     rpc_version = nsxv3_constants.NSXV3_SERVER_RPC_VERSION
 
-    _LIMIT = 100
-    _CREATE_AFTER = datetime.utcfromtimestamp(0).isoformat()
-
-    def __init__(self, context, topic, host):
-        target = oslo_messaging.Target(topic=topic, version=self.rpc_version)
+    def __init__(self):
+        target = oslo_messaging.Target(topic=nsxv3_constants.NSXV3_SERVER_RPC_TOPIC, 
+                                       version=self.rpc_version)
+        self.context = neutron_context.get_admin_context()
         self.client = rpc.get_client(target)
-        self.context = context
-        self.host = host
-        self.topic = topic
+        self.host = cfg.CONF.host        
 
     @log_helpers.log_method_call
-    def get_port_revision_tuples(
-            self, limit=_LIMIT, created_after=_CREATE_AFTER):
+    def get_ports_revisions(self, limit, offset):
         cctxt = self.client.prepare()
-        return cctxt.call(
-            self.context,
-            'get_port_revision_tuples',
-            host=self.host,
-            limit=limit,
-            created_after=created_after)
+        return cctxt.call(self.context, 'get_ports_revisions',
+                          host=self.host, limit=limit, offset=offset)
 
     @log_helpers.log_method_call
-    def get_qos_policy_revision_tuples(
-            self, limit=_LIMIT, created_after=_CREATE_AFTER):
+    def get_qoses_revisions(self, limit, offset):
```

Comment:
I am not a fan of `qoses`, and would suggest to rather to stay with `qos_policies`. Here I also would like some word marking the relationship between QoS-Policies and revisions.